src/lang/po/de/Prusa-Firmware-Buddy_de.po

```diff
@@ -883,7 +883,7 @@ msgstr "USB-Laufwerk schreibgeschützt!"
 
 #: src/gui/screen_print_preview.cpp:162
 msgid "Used Filament"
-msgstr "Genutztes Filament"
+msgstr "Verbrauchtes Filament"
```

Comment:
Original and new are correct, as "nutzen/benutzt" is used more often the original makes bit more sense.

Reply:
But anything that is supposed to be used up, like stock metal, drill bits etc. is called
Verbrauchsmaterial in German, so Verbrauchen is the correct word. 
I can "benutzen"-use my bike every day and it won't be gone
But I can not do that with filament.